rca/project_styleguide/templates/patterns/pages/programmes/programme_index.html

```diff
@@ -2,42 +2,59 @@
 {% load wagtailcore_tags wagtailimages_tags navigation_tags static %}
 
 {% block content %}
-<div class="wrapper">
-    <section class="section section--main">
-        <h1>{{ page.title }}</h1>
-        <p>{{ page.introduction }}</p>
-
-        {% if subpages %}
-            <div class="grid">
-                {% if subpages.object_list.exists %}
-                    {% for subpage in subpages.object_list.specific %}
-                        <a class="card-with-image" href="{% pageurl subpage %}">
-                            {% if subpage.listing_image %}
-                                {% image subpage.listing_image fill-450x300 %}
-                            {% else %}
-                                {# Add a default image here #}
-                            {% endif %}
-                            <div class="card">
-                                <h3 role="listitem">
-                                    {% firstof subpage.listing_title subpage.title %}
-                                </h3>
-                                {% if subpage.listing_summary or subpage.introduction %}
-                                    <p>{% firstof subpage.listing_summary subpage.introduction %}</p>
-                                {% endif %}
-                            </div>
-                        </a>
-                    {% endfor %}
-                {% else %}
-                    {# no items on this page #}
-                {% endif %}
+    <div class="page">
+
+        <header class="page__header bg bg--dark">
+            <div class="title-area grid">
+                <div class="title-area__content">
+                    <h1 class="title-area__header heading heading--display-two">{{ page.title }}</h1>
+                    <p class="title-area__introduction introduction">{{ page.introduction|richtext }}</p>
+                </div>
             </div>
+        </header>
+
+        <input type="search" placeholder="{{ page.search_placeholder_text }}"/>
+
+        <div class="page__content">
+
+            {% if subpages %}
+                <div class="grid">
+                    {% if subpages.object_list.exists %}
+                        {% for subpage in subpages.object_list.specific %}
+                            <a class="card-with-image" href="{% pageurl subpage %}">
+                                {% if subpage.hero_image %}
```

Comment:
Do we need to work in the listing_image here? Usually, I use the listing image if specified, but if not use the hero

Reply:
According to https://projects.torchbox.com/projects/rca-website-rebuild/tickets/249 we’re meant to use the hero image for each programme, although I haven’t checked whether all programmes do have a hero image defined.

This will only be relevant when fetching this data from the API though (this listing in the template was just a placeholder).